### PR TITLE
Fix line info with multiline strings

### PR DESCRIPTION
**Pull request by hellerve**
This PR fixes the line information that is reported when multiline strings are used in a program. Prior to this PR, the linebreaks in a string did not increment the count in the parser state, such that any reporting after them was inaccurate and often unusable. This PR should fix it.

You can try it out with a program like this, for instance:

```clojure
(def ml-string "hi
i span
multiple
lines")

(defn main [] (+ 1 invalid))
```

The parser currently in master reports an error on line 3, whereas this patch makes the parser realize that the error is actually on the last line.

Cheers

### CODE REVIEW

File: src/Parsing.hs [hunk 1]
```diff
@@ -254,6 +256,17 @@ incColumn x = do s <- Parsec.getState
                  Parsec.putState (s { parseInfo = newInfo })
                  return ()
 
+incLine :: Int -> Parsec.Parsec String ParseState ()
```
[the remark this answers is not in the document]
Comment (hellerve) [reply src/Parsing.hs]: This is basically an unrolled generalization of `linebreak` (see also `incColumn` above). We can also define `linebreak` in terms of this function.

[the remark this answers is not in the document]
Comment (eriksvedang) [reply src/Parsing.hs]: Sorry for the sloppy code review, this is totally fine!

**Comment by eriksvedang** [comment]
Good stuff!

File: src/Parsing.hs (hunk 1 above)
[the remark this answers is not in the document]
Comment (hellerve) [reply src/Parsing.hs]: Oh no, you’re good :) thanks for asking

**Pull request merged**